repoarchivetool/app.py

```diff
@@ -9,11 +9,17 @@
 import authentication_interface
 
 archive_threshold_days = 30
-organisation = "ONS-Innovation"
 
 app = flask.Flask(__name__)
 app.config['SECRET_KEY'] = os.urandom(24)
 
+# Set variables from the environment
+account = os.getenv("AWS_ACCOUNT_NAME")
```

Comment:
Where these are pulled from the running environment, won't instructions for running outside a container be different? The user would have to provide a .env file and that'll have to be loaded into the environment.

Might be easier to remove local running instructions and force running in a container only?

Reply:
os.getenv should just pickup the value if you have configured it locally before running the app
i.e export AWS_ACCOUNT_NAME=xyz

Do you want me to add something to the readme?

Reply:
Added to README